etl/churn.kp/knowledge.md

```diff
@@ -428,12 +428,16 @@ def upload_to_s3(records, churn_outfile):
     transfer.upload_file(churn_outfile, churn_bucket, churn_s3,
                          extra_args={'ACL': 'bucket-owner-full-control'})
 
-    # Also upload it to the dashboard:
-    # Update the dashboard file
-    dash_bucket = "net-mozaws-prod-metrics-data"
-    dash_s3_name = "telemetry-churn/{}".format(churn_outfile)
-    transfer.upload_file(churn_outfile, dash_bucket, dash_s3_name,
-                         extra_args={'ACL': 'bucket-owner-full-control'})
+    # TODO: Re-enable uploading to the dashboard when cutover happends
```

Comment:
I will be disabling uploads to the dashboard while the csv location is still under `amiyaguchi/churn`. Once we verify that everything looks good, we can disable the ATMO script and enable this.